### PR TITLE
Multiple fixes

### PR DESCRIPTION
**Pull request by JbIPS**
* Service listing
* Display name in breadcrumb
* UnifileService optimization

### CODE REVIEW

**Comment by JbIPS** [comment]
Bump

File: src/sass/style.scss [hunk 1]
```diff
@@ -13,9 +13,9 @@ $button-hover: $active-bg; //#31bbff;
 
 // fonts
 $fa-font-path: "fonts";
-@import "../../node_modules/font-awesome/scss/font-awesome.scss";
```
Comment (lexoyo) [inline src/sass/style.scss]: This doesn't work on my local test

Comment (JbIPS) [reply src/sass/style.scss]: How are you testing (inside a project, inside CE)?

Comment (JbIPS) [reply src/sass/style.scss]: I just found out something that could solve our problems: https://github.com/stackcss/sheetify#use-npm-packages

**Pull request closed without merging**